source/mainnet/net/guides/run-node-macos.rst

```diff
@@ -258,6 +259,31 @@ If you want to change whether the node services start automatically, you have tw
 
          $sudo rm "/Library/LaunchDaemons/software.concordium.testnet.node-collector.plist"
 
+.. _node-collector-configuration-macos:
+
+Node collector configuration
+============================
+
+Since version 5.3.0 of the node, the collector uses the GRPC V2 interface. Therefore, in order to run the collector, it is required that the node which the collector connects to has the GRPC V2 interface enabled.
```

Comment:
This should ideally be 5.3.2, or perhaps we can just say `5.3.*` to avoid updating it.